### PR TITLE
pd_client: fix item value type

### LINKED ISSUE #14104: pd_client:  provide bytes struct to global item value
## Development Task
The proto of GlobalConfigItem is a string, which should be bytes.
Because protobuf has an invalid string check. Due to #13983, we make value support other structures that are not a valid utf8 string and therefore should provide bytes

### PR DESCRIPTION
**Pull request by HuSharp**
Signed-off-by: husharp <jinhao.hu@pingcap.com>

### What is changed and how it works?

Issue Number: Close #14104 Should merge after https://github.com/pingcap/kvproto/pull/1051

What's Changed:

```commit-message
We need to use the new field to support item value as bytes to avoid proto string check failures.
```

### Check List 

Tests 

- Unit test
- Integration test

### Release note 

```release-note
None
```

### CODE REVIEW

File: components/resource_control/src/resource_group.rs [hunk 1]
```diff
@@ -41,14 +41,9 @@ pub struct ResourceGroupManager {
 impl ResourceGroupManager {
     fn get_ru_setting(rg: &ResourceGroup, is_read: bool) -> u64 {
         match (rg.get_mode(), is_read) {
-            (GroupMode::RuMode, true) => rg
+            (GroupMode::RuMode, _) => rg
```
Comment (glorv) [inline components/resource_control/src/resource_group.rs:44]: We can remove the parameter `is_read` now

**Review by glorv — COMMENTED** [review]
rest LGTM

File: components/resource_control/src/resource_group.rs (hunk 1 above)
Comment (HuSharp) [reply components/resource_control/src/resource_group.rs:44]: keep it to consider `RawMode`

**Review by glorv — APPROVED** [review]
LGTM

**Review by BornChanger — APPROVED** [review]

**Review by Connor1996 — APPROVED** [review]
LGTM

**Pull request merged**